docs/content/examples/alt-text.html

```diff
@@ -0,0 +1,152 @@
+            <tr>
+                <td>“Illustration of a rocket.”</td>
+            </tr>
+            <tr>
+                <td>Phoot of a rocket.”</td>
+            </tr>
+            <tr>
+                <td>“Icon of a rocket.”</td>
+                <td>“A rocket, icon.”</td>
+            </tr>
+        </tbody>
+    </table>
+    {% header h3 | End with a period %}
+    <p class="stacks-copy">
+        End the alt-text with a period. This makes screen readers pause a bit after the last word in the alt-text, creating a natural pause before the next bit of text.
+    </p>
+</section>
+<section class="stacks-section">
+    {% header h2 | When not to write one %}
+    <p class="stacks-copy">
+        In most cases you should use an alt-text for images, but there are some exceptions where you should leave the alt-text blank.
+    </p>
+    {% header h3 | Decorative images %}
+    <p class="stacks-copy">
+        If an image does not convey any meaning to the user, leave the alt-text blank.
+    </p>
+    <div class="stacks-preview mb16">
+{% highlight html %}
+<div>
+    <img src="rocket.svg" alt="">
+    <h2>…</h2>
+    <p>…</p>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid fd-column gsy gs8 wmx3">
+                <div class="grid--cell">
+                    <img src="{{ "/assets/img/efficiency.svg" | relative_url }}"" alt="">
+                </div>
+                <h2 class="grid--cell fs-body3 fw-normal fc-dark">Improve efficiency and ship better code</h2>
+                <p class="grid--cell fs-body1 fc-medium m0">There’s a reason the world’s largest developer teams rely on Stack Overflow Enterprise—it leads to better product and allows distributed teams to securely collaborate and share knowledge.</p>
+            </div>
+        </div>
+    </div>
+    <p class="stacks-copy">
+        The rocket here doesn’t add meaningful information.
+    </p>
+    {% header h3 | Images accompanied by text %}
+    <p class="stacks-copy">
+        If an image has a label nearby, leave the alt-text blank.
+    </p>
+    <div class="stacks-preview mb16">
+{% highlight html %}
+<div>
+    <img src="leaderboard.svg" alt="">
+    <h2>…</h2>
+    <p>…</p>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid gsx gs8 ai-center wmx4">
+                <div class="grid--cell">
+                    <img src="{{ "/assets/img/leaderboard.svg" | relative_url }}"" alt="" width="44" height="43">
+                </div>
+                <div class="grid--cell">
+                    <h2 class="grid--cell fs-body1 fw-bold fw-normal fc-dark mb4">TOP LEADERBOARD: 728X90</h2>
+                    <p class="grid--cell fs-body1 fc-medium m0">This ad unit is the most visible on the site.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <p class="stacks-copy">
+        The nearby text here already explains what the graphic illustrates.
```

Comment:
We can maybe add why. 

For example, changing "graphic illustrates." to "graphic illustrates. If there was alt-text in this scenario, screen readers, for example, would read nearly duplicate information to the user."